src/components-barnes-toolkit/scss/atoms/_logo.scss

```diff
@@ -86,46 +85,71 @@ a.a-logo {
   fill: currentColor;
 }
 
-.a-logo__svg--s {
+.a-logo__svg--xs {
   display: block;
 
-  @include breakpoint('medium+') {
+  @include breakpoint("small+") {
+    display: none;
+  }
+}
+
+.a-logo__svg--s {
+  display: none;
+
+  @include breakpoint("small+") {
+    display: block;
+  }
+
+  @include breakpoint("medium+") {
```

Comment:
Adding in the styling for the xs breakpoint here removes the icon at xs screen sizes with the standard logo!

I would instead nest this logic inside `.a-logo__svg--100` so it will only apply to the 100 logo.